devel/go-jsonnet/Portfile

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/google/go-jsonnet 0.17.0 v
```

Comment:
Looks good.  Let's just add a `revision` field and this is good to go:
```
go.setup            github.com/google/go-jsonnet 0.17.0 v
revision            0
```

Reply:
Done. Thanks for reviewing.